### PR TITLE
Pvp 104 add sport personal task creation form

### PR DESCRIPTION
**Pull request by youreondrugs**

### CODE REVIEW

**Review by vytjar — CHANGES_REQUESTED** [review]
Not all fields that are needed to create a SportTask (model -> Tasks.kt -> SportTask) are covered.

File: app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt [hunk 1]
```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
...
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            //horizontalAlignment = Alignment.CenterHorizontally
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt:66]: If horizontal alignment is not required, then this redundant comment should be removed. Else, uncomment it to make it apply.

File: app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt [hunk 2]
```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
...
+        ){
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt:67]: Line :67 requires a single space before curly brackets. It should automatically be generated by using auto-formatting of your IDE. By default, the keybind should be `Ctrl+Alt+L`.

File: app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt [hunk 3]
```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
...
+            ) {
+            Checkbox(
+                checked = supportsDistanceMetrics,
+                onCheckedChange = { supportsDistanceMetrics = it },
+                modifier = Modifier.padding(bottom = 16.dp)
+            )
+            Text (
+                text = "Supports distance metrics",
+                modifier = Modifier.padding(start = 16.dp)
+            )}
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt:80]: Line :77 has redundant space before opening brackets.
Separate components should be separated by a single empty line to increase readability.
Wrong padding, both `Checkbox` and `Text` components are within `Row` component's scope, you should increase their padding by 4 spaces (single TAB).

File: app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt [hunk 4]
```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
...
+                value = day.toString(),
+                onValueChange = { },
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                modifier = Modifier.weight(1f)
+            )
+        }
+    }
+}
+
+
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt:154]: Redundant empty line, remove one out of the two.

File: app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt [hunk 5]
```diff
@@ -15,4 +44,116 @@ fun TaskScreen(
...
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview() {
+    SportTaskForm()
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt:158]: I have a feeling that this component is unused. If that is the case (and it is only used for previewing purposes), then it should be removed before executing a merge. You can preview `SportTaskForm` component by annotating it with `@Preview` annotation. This annotation can be given to multiple components within the same file, all should be rendered in the IDE side just fine.

Also, at the end of the file a single empty line _could_ be added for consistency. That's how UNIX file endings work.

**Review by klaidoshka — CHANGES_REQUESTED** [review]

**Review by vytjar — APPROVED** [review]

**Review by klaidoshka — APPROVED** [review]

**Pull request merged**